grafana-plugin/src/containers/UserSettings/parts/tabs/MattermostInfo/MattermostInfo.module.css

```diff
@@ -0,0 +1,13 @@
+.footer {
```

Comment:
Can we pls use Emotion for styling? We moved from scss/css modules.

You can find some example here: https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/containers/UserSettings/parts/tabs/TelegramInfo/TelegramInfo.tsx#L115

Reply:
I have made the changes for the current PR. I have also raised a separate PR for other files in the base branch https://github.com/grafana/oncall/pull/5103/files